ie_class/plugins_apple_arm.xml

```diff
@@ -0,0 +1,11 @@
+<!-- 
+Copyright (C) 2020 Intel Corporation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<ie>
+    <plugins>
+        <plugin location="libopenvino_arm_cpu_plugin.so" name="CUSTOM">
```

Comment:
Why do we need this plugins.xml with strange name? Can we register plugin in the runtime or at least use the normal name?